index.html

```diff
@@ -1,22 +1,237 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<meta charset="utf-8">
-	<title></title>
+	<title>YouTube Video Player</title>
+	<script src="https://kit.fontawesome.com/11bb635b00.js" crossorigin="anonymous"></script>
+	<style>
+		@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+	</style>
+	<link rel="stylesheet" href="styles.css">
 </head>
+
 <body>
-<div id="nav-bar"></div>
-<div id="video-player">
-	<div id="video-screen"></div>
-	<div id="video-title"></div>
-	<div id="video-description"></div>
-</div>
-<div id="thumbnails">
-	<div id="thumbnail-one"></div>
-	<div id="thumbnail-two"></div>
-	<div id="thumbnail-three"></div>
-	<div id="thumbnail-four"></div>
-	<div id="thumbnail-five"></div>
-</div>
+	<section id="navbar">
+		<div id="hamburger-button">
+			<button class="hamburger">
```

Comment:
The `button` should always have a `type` attribute to tell browsers what type of button it is. Please keep this in mind.